### PR TITLE
Docker for Desktop updates

### PR DESCRIPTION
**Pull request by akimd**
### Proposed changes

This is mostly to prepare the documentation to receive updates.  There's a lot of work needed in this area of the documentation, and it's hard to break in small PRs as they all depend on each other.

- clean up markdown
- remove details about obsolete versions of Docker for Desktop
- update some images
- update some items

Prompted by https://github.com/docker/desktop-experience-backlog/issues/37.

cc @gtardif

### CODE REVIEW

**Comment by gtardif** [comment]
cc @gbarr01

**Comment by gbarr01** [comment]
Thanks, @gtardif , I am actually working on this PR right now and will push some extra changes soon.

**Comment by gbarr01** [comment]
Hi, @akimd ... I did more updating (but there is more to do): 

- updated more images (reduced size, removed shadow)
- tightened some of the text (too many words on this page)
- updated experimental version code
- repaired some links. 

Let me know if you disagree with anything.

File: docker-for-mac/index.md [hunk 1]
```diff
@@ -90,191 +90,147 @@ docker-machine version {{ site.machine_version }}, build 9ba6da9
...
+  manually by choosing ![whale menu](/docker-for-mac/images/whale-x.png){: .inline} -> **Check for Updates**.
 
-- Check **Include VM in Time Machine backups** to back up the Docker for Mac
-  virtual machine. (By default, this is unchecked.)
+- **Include VM in Time Machine backups** backs up the Docker for Mac virtual machine. (Disabled by default.)
 
-- You have the option to **Securely store Docker logins in MacOS keychain**,
-  enabled by default. To "opt out" of storing your Docker login credentials,
-  uncheck this option.
+- **Securely store Docker logins in MacOS keychain** stores your Docker login credentials. (Enabled by default.)
```
Comment (akimd) [inline docker-for-mac/index.md]: I believe the official spelling is macOS.

**Comment by akimd** [comment]
@gbarr01 For the records, you did a wonderful job!  I believe reading your changes will help me provide a better initial version of forthcoming changes.  Thanks a lot!

**Pull request merged**